doc/geany.txt

```diff
@@ -3624,6 +3624,12 @@ Reflow lines/block                                        Reformat selected line
                                                           breaking lines at the long line marker or the
                                                           line breaking column if line breaking is
                                                           enabled for the current document.
+
+Join Lines                                                If no selection, remove line endings and indentation
+                                                          after the current line until the end of the current
+                                                          (indented) text block.
```

Comment:
I'm not sure if this is entirely clear that it means "the block of lines at the current indentation level, separated by blank lines (indented or not)".

Reply:
Thanks, updated.